pkg/domain/entities/events.go

```diff
@@ -64,6 +68,7 @@ func ConvertToEntitiesEvent(e libpodEvents.Event) *types.Event {
 		attributes["containerExitCode"] = strconv.Itoa(*e.ContainerExitCode)
 	}
 	attributes["podId"] = e.PodID
+	attributes["network"] = e.Network
```

Comment:
this should be wrapped in `if e.Network != "" {}` to only add this attribute if there is a network.
Note to self I have not checked how this is exposed in the docker compat API.